lib/model/message_list.dart

```diff
@@ -937,23 +937,20 @@ class MessageListView with ChangeNotifier, _MessageSequence {
     final generation = this.generation;
     bool hasFetchError = false;
     try {
-      final GetMessagesResult result;
-      try {
-        result = await getMessages(store.connection,
-          narrow: narrow.apiEncode(),
-          anchor: anchor,
-          includeAnchor: false,
-          numBefore: numBefore,
-          numAfter: numAfter,
-          allowEmptyTopicName: true,
-        );
-      } catch (e) {
-        hasFetchError = true;
-        rethrow;
-      }
+      final result = await getMessages(store.connection,
+        narrow: narrow.apiEncode(),
+        anchor: anchor,
+        includeAnchor: false,
+        numBefore: numBefore,
+        numAfter: numAfter,
+        allowEmptyTopicName: true,
+      );
       if (this.generation > generation) return;
 
       processResult(result);
+    } catch (e) {
+      hasFetchError = true;
```

Comment:
```
msglist [nfc]: Remove one nested try block in `_fetchMore`

The nested `try` block doesn't seem to be making any difference,
so good to remove.
```

This isn't NFC: now, `hasFetchError` is about what happens when data is fetched _and processed_ (with `processResult`), not just about what happens when data is fetched. That doesn't seem desirable, just from looking at its name.